### DISCUSSION TITLE
how to get KafkaCompanionTestBase?

### DISCUSSION THREAD

**Discussion by PyAntony** [smallrye/smallrye-reactive-messaging#1701] [Q&A] [answered]
Hi,

I am trying to use the KafkaCompanionTestBase as indicated here (https://smallrye.io/smallrye-reactive-messaging/3.15.0/kafka/test-companion/) but it doesn't work, I can't find the base class. Documentation does say that this is a different library so what am I suppose to put in the pom file? the following doesn't work:

    <dependency>
      <groupId>io.smallrye.reactive</groupId>
      <artifactId>smallrye-reactive-messaging-kafka-test-companion</artifactId>
      <scope>test</scope>
    </dependency>

**Comment by ozangunalp** [comment, marked as answer]
If you are using Quarkus I think it is only available from the bom starting from `2.8.0.CR1`. 
BTW you can also use the KafkaCompanion test resource dependency, which will also pull `smallrye-reactive-messaging-kafka-test-companion`:
```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-test-kafka-companion</artifactId>
            <scope>test</scope>
        </dependency>
```

**Reply by PyAntony** [reply]
Just switching to Quarkus 2.8 didn't work, but adding the test resource dependency worked. Thanks.
